samples/enumdevdir.c

```diff
@@ -16,31 +16,47 @@
 #include <cc65.h>
 
 
-void printdir (char *newdir)
+int printdir (char *newdir)
```

Comment:
Can you please make use of https://github.com/cc65/cc65/blob/master/include/stdbool.h ?
Success -> true
Failure -> false